cmd/gaia/app/app.go

```diff
@@ -183,6 +203,32 @@ func NewGaiaApp(logger log.Logger, db dbm.DB, traceStore io.Writer, loadLatest b
 	return app
 }
 
+// CheckTransferDisabledBurnMultiSend
+func CheckTransferDisabledBurnMultiSend(msg bank.MsgMultiSend) bool {
```

Comment:
can't we just require the burn output to be the first?

Reply:
i think we can...

Reply:
I'm not sure.  Is there any rules about how the inputs get ordered, do they have to be sorted or something?

Reply:
No, there are no rules about the order of the inputs (the coins within the inputs must be sorted).

Reply:
looks like this needs follow up